### PR TITLE
VPN is no longer required (except for UA login nodes)

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: intro-HPC/ch_preparing_the_environment.tex [hunk 1]
```diff
@@ -189,7 +189,7 @@ \section{First Time connection to the \hpc}
   \subsection{Connect}
   \label{sec:connect}
 
-  The \hpc is only accessible from within the \university network, but you can
+  At some institutes the \hpc is only accessible from within the \university network, but you can
```
Comment (wpoely86) [inline intro-HPC/ch_preparing_the_environment.tex]: Is this still true? I know for us  (VUB) it is not the case, Leuven neither. Gent the same? I think only Antwerp might be? @klust ?

Comment (JensTimmerman) [reply intro-HPC/ch_preparing_the_environment.tex]: Hence the change in wording here.
I wanted to add an `\ifantwerp`, but it seems to break inside an 
`\ifmacorlinux`

**Comment by JensTimmerman** [comment]
@wpoely86 any way of fixing `The job exceeded the maximum time limit for jobs, and has been terminated.`
Faster/smaller latex install?

**Comment by wpoely86** [comment]
Yes, smaller install or try the cache in Travis.

What we can do is do only a very basic install of LaTeX and install just the package we use on top of that.

**Comment by boegel** [comment]
@JensTimmerman Please sync with `master`, re-bump version so we can try again?

**Pull request merged**